platformio.ini

```
@@ -11,11 +11,14 @@
 platform = espressif32@1.11.1
 board = pico32
 framework = espidf
+build_flags =
+	-DBUILD_TIME=$UNIX_TIME
+	!python3 -c 'import subprocess; rev = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).strip(); print("-DGIT_COMMIT_ID_SHORT=\\\"{}\\\"".format(rev.decode("ASCII")))'
```

```suggestion
	!sh -c "echo -DGIT_COMMIT_ID_SHORT=$(git rev-parse --short HEAD)"
```

why use python, instead of simple shell call?

---

As discussed in person, this is the most reliable and convenient way